### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by voiceofvraja** [TeamNewPipe/NewPipe#5201]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Ability to remove like/dislike buttons if user want just like comments removal.

#### How will you/everyone benefit from this feature?

We all know newpipe is not for logging in and saving everything, so the like button is kind of useless there. One use is to usee how many like how many do not but it still feels like it takes up extra space there and you could just add the views count which is already there and remove or give us the option to remove the like buttons just like we can remove the comments. Or you can just change it a bit like...

> 33% viewers liked this video
> 33/67 is the like/dislike ratio

**Comment by c-shubh** [comment]
Instead of removing the like/dislike buttons, it better be repurposed as said in #5159

**Comment by voiceofvraja** [comment]
NO. As I said it is user's choice and that is why I opened this request

**Comment by HTSamurai** [comment]
Or maybe an option between the two?

**Comment by voiceofvraja** [comment]
What you mean? @HTSamurai

**Comment by HTSamurai** [comment]
> What you mean? @HTSamurai

I mean an option fromworking like buttons and your like to dislike ratio/no likes idea

**Comment by voiceofvraja** [comment]
Oh, that is on the dev then.

**Comment by mhmdanas** [comment]
@vrindavan to be honest, I don't see how this is necessary. You said that it should be possible to hide them just like comments, but comments take up a whole tab on their own, while these two "buttons" don't take up much estate. While I could see value in allowing complete customization of the video page (which is another request in and of itself), having a setting just for this sounds overkill to me.

**Comment by vrindavan** [comment]
Yeah, whatever you find better. I don't mind any anymore.
